### ISSUE TITLE
Custom axes titles for learning curve subplots

### ISSUE THREAD

**Issue by VinodKumar9576** [DistrictDataLabs/yellowbrick#1064]
Please add options to add custom x, y axes labels when plotting subplots for learning curves.
At present unable to add them. Also same goes with xticks, yticks.

Xticks: As you can see for my implementation there's no fixed xticks for each subplots. Make an option to add custom xticks.

![image](https://user-images.githubusercontent.com/23068132/81775905-54e9aa80-950b-11ea-87f0-8eef81084147.png)

**Comment by VinodKumar9576** [comment]
@bbengfort Can you please look into it.

**Comment by rebeccabilbro** [comment]
Hi @VinodKumar9576 and thanks for checking out Yellowbrick. To achieve the behavior you're looking for, you can modify the axes directly, as with all Matplotlib plots:

```python
import matplotlib.pyplot as plt 

from sklearn.tree import DecisionTreeRegressor
from sklearn.model_selection import ShuffleSplit

from yellowbrick.datasets import load_energy
from yellowbrick.model_selection import LearningCurve


X, y = load_energy()
depths = [1, 3, 6, 10]

fig, axes = plt.subplots(2, 2)
fig.set_size_inches(20, 16)

for idx, ax in enumerate(axes.flatten()):
    depth = depths[idx]
    model = DecisionTreeRegressor(max_depth=depth)
    visualizer = LearningCurve(
        model,
        scoring="r2",
        cv=ShuffleSplit(n_splits=10, test_size=0.2, random_state=0),
        ax=ax
    )
    visualizer.fit(X, y)
    visualizer.finalize()
    ax.set_title(f"Learning curve with depth: {depth}")
    ax.set(xlabel="custom x", ylabel="custom y")

    ax.set_xticks([50, 150, 250, 350, 450, 550, 650])

plt.show()
```

<img width="898" alt="Screen Shot 2020-05-13 at 9 20 52 AM" src="https://user-images.githubusercontent.com/8760385/81817702-16e48a80-94fb-11ea-9e85-79bee7aa57c6.png">
